src/SDKs/SqlManagement/Sql.Tests/ManagedDatabaseVulnerabilityAssessmentTestFixture.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Azure.Management.ResourceManager.Models;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using Sku = Microsoft.Azure.Management.Sql.Models.Sku;
+
+namespace Sql.Tests
+{
+    public class ManagedDatabaseVulnerabilityAssessmentTestFixture : IDisposable
```

Comment:
Since this is just creating a Managed Instance, consider renaming this to `ManagedInstanceTestFixture`

Reply:
I renamed it